pkg/git/github.go

```diff
@@ -49,6 +57,119 @@ func DeriveRepositoryFullnameFromUrl(githubRepoUrl string) (RepositoryFullname,
 	}, nil
 }
 
+type CoreCtlAsset struct {
```

Comment:
I would say we shouldn't keep this logic in `git` package and we should move near the update cmd, since it's very logic to corectl udpate